docs/elements.md

```diff
@@ -583,7 +583,7 @@ Attribute | Type | Default | Description
 --- | --- | --- | ---
 `answers-name` | string | — | Variable name to store data in.
 `weight` | integer | 1 | Weight to use when computing a weighted average score over elements.
-`correct-answer` | string | special | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`.
+`correct-answers` | string | special | Correct answer for grading. Defaults to `data["correct_answers"][answers-name]`.
```

Comment:
Ditto

Reply:
Same for this one, the string input questions expect `correct-answers`

<img width="626" alt="Screen Shot 2021-05-08 at 2 16 48 PM" src="https://user-images.githubusercontent.com/2507459/117553744-14324d80-b008-11eb-9ff0-aa397e48fb86.png">